connectors/routing/connector.go

```diff
@@ -44,46 +42,21 @@ type RoutingConnector struct {
 	// for the value of Config name prefix, strict string without "*" always comes first,
 	// and then string with "*" suffix (glob match) and pure "*".
 	// There shouldn't be any scope with a prefix "*" like "*.service.v1"
-	CConfigs []*ConnectorConfig
+	config Config
+	connectors map[string]dosa.Connector
 	// PluginFunc is a plugin that passes in
 	// the scope, namePrefix and operation name, returns wanted scope and namePrefix
 	PluginFunc PluginFunc
 }
 
 // NewRoutingConnector initializes the RoutingConnector
 // connectorMap has a key of connectorName, and the value is a dosa.connector instance
-func NewRoutingConnector(cfg Config, connectorMap map[string]dosa.Connector, plugin PluginFunc) (*RoutingConnector, error) {
-	cConnectors, err := NewConnectorConfigs(cfg, connectorMap)
-	if err != nil {
-		return nil, errors.Wrap(err, "could not initialize ConfiguredConnectors")
-	}
-	connSlice := sortConnectorSlice(cConnectors)
-
+func NewRoutingConnector(cfg Config, connectorMap map[string]dosa.Connector, plugin PluginFunc) *RoutingConnector {
```

Comment:
since it's in routing/connectors.go, shall we just change it to NewConnector()?

Reply:
+1